drivers/wifi/nrf700x/zephyr/src/qspi/src/qspi_if.c

```diff
@@ -1046,7 +1070,19 @@ int qspi_WRSR2(const struct device *dev, uint8_t data)
 
 int qspi_cmd_wakeup_rpu(const struct device *dev, uint8_t data)
 {
-	return qspi_WRSR2(dev, data);
+	int ret;
+	/* Use lowest 8MHz for waking up RPU */
```

Comment:
Aren't we supposed to use 12 MHz when RPU is sleeping?

Reply:
12MHz is the maximum we can go but as its just waking up we don't need to use the absolute max, 8MHz is best bet across boards.

Reply:
@udaynordic: Do you think this should be ok?

Reply:
We can stick to 8MHz instead of Max frequency.